<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Terence S. Crofts</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
    <link href='https://fonts.googleapis.com/css?family=Roboto:300,400,700' rel='stylesheet' type='text/css'>
    <link href="style.css" type="text/css" rel="stylesheet">
    <link rel="preconnect" href="https://fonts.gstatic.com">
<link href="https://fonts.googleapis.com/css2?family=Questrial&display=swap" rel="stylesheet">
<link rel="preconnect" href="https://fonts.gstatic.com">
<link href="https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,300;0,400;0,500;0,600;0,700;1,300;1,400;1,500&display=swap" rel="stylesheet">
  </head>
  <body>

    <header>
      <div>
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
          <div class="topnav" id="myTopnav">
            <a href="index.html">Home</a>
            <a href="projects.html">Research</a>
            <a href="publications.html">Publications</a>
            <a href="contact.html">Contact</a>
            <a href="javascript:void(0);" class="icon" onclick="myFunction()">
    <i class="fa fa-bars"></i>
  </a>
</div>
<script>
// When the user scrolls the page, execute myFunction
window.onscroll = function() {myFunction()};

// Get the navbar
var myTopNav = document.getElementById("myTopnav");

// Get the offset position of the navbar
var sticky = myTopNav.offsetTop;

// Add the sticky class to the navbar when you reach its scroll position. Remove "sticky" when you leave the scroll position
function myFunction() {
  if (window.pageYOffset >= sticky) {
    myTopnav.classList.add("sticky")
  } else {
    myTopnav.classList.remove("sticky");
  }
}
</script>
      </div>
    </header>

    <section class="jumbotron">
      <div class="container">
        <div class="row">
          <h2>Research</h2>
        </div>
      </div>
    </section>

    <section>
      <div>
        <div>
          <p><i>My research characterizes the evolution and dynamics of interactions between emerging contaminants – such as antibiotics or pharmaceuticals – and organisms in microbial communities. These interactions encompass modification (e.g. xenobiotic detoxification, antibiotic resistance), degradation (e.g. catabolism, mineralization), and community restructuring (e.g. dysbiosis)</p>
          </i>
          <img src="Resources/images/210418_ProjectsSummary_v1.jpg" class="center"/>
          <p align="center"> I focus on three lines of work that utilize parallel approaches to directly study novel interactions between small molecules and organisms in microbial communities: Mechanism and evolution of antimicrobial modification and degradation, Gut microbiota-mediated modification of pharmaceuticals and metabolites, Microbial community responses to emerging contaminants. I approach these lines of work from complementary angles and at multiple levels of scrutiny (i.e. spanning metagenomics to biochemistry). By studying these functions in isolation and in their native context it is possible to gain greater insight into their evolutionary origins and biochemical niche, as well as to access the great genetic diversity found in microbiomes.</p>
        </div>
      </div>
    </section>

<section>
    <button type="button" class="collapsible"><h4>Antimicrobial Catabolism and Resistance</h4></button>
<div class="content">

<h5 align="center">Description</h5>
  <p>Most clinically-relevant antimicrobials are natural products of soil-dwelling bacteria. The soil harbors diverse microbes which can resist or degrade antibiotics and, in some cases, even catabolize them. While resistance genes are known to be ancient and widely-distributed in the soil, and have been deeply mechanistically characterized, there is a dearth of equivalent knowledge and characterization of mechanisms of antibiotic catabolism. Given its linkage to resistance, understanding the mechanisms of antibiotic catabolism is a pressing concern. Discovery and characterization of these mechanisms and their evolvution and regulation can teach us about the ecology of antibiotics in the environment. Elucidation of antibiotic catabolism pathways may allow for bioremediation of antibiotic-contaminated soils and discovery of antibiotic-remodeling enzymes with utility to industry.</p>
  <h5 align="center">Publications</h5>
<ul>
<li><b>Crofts TS</b>, Sontha P, King AO, Wang B, Biddy BA, Zanolli N, Gaumnitz J, Dantas G (2019) <b>Discovery and characterization of a nitroreductase capable of conferring bacterial resistance to chloramphenicol</b>. Cell Chemical Biology. 26 (4). 559-70.<a href="Resources/Paper PDFs/Crofts et al. - 2019 - Discovery and characterization.pdf" target="_blank">[PDF]</a></li>

<li><b>Crofts TS</b>, Wang B, Spivak A, Gianoulis TA, Forsberg KJ, Gibson MK, Johnsky LA, Broomall SM, Rosenzweig CN, Skowronski EW, Gibbons HS, Sommer MOA, Dantas G (2018) <b>Shared strategies for β-lactam catabolism in the soil microbiome</b>. Nature Chemical Biology, 14 (6). 556-64.<a href="Resources/Paper PDFs/Crofts et al. - 2018 - Shared strategies for β-lactam catabolism in the soil microbiome.pdf" target="_blank">[PDF]</a></li>

<li><b>Crofts TS</b>, Wang B, Spivak A, Gianoulis TA, Forsberg KJ, Gibson MK, Johnsky LA, Broomall SM, Rosenzweig CN, Skowronski EW, Gibbons HS, Sommer MOA, Dantas G (2017) <b>Draft genome sequences of three β-lactam-catabolizing soil Proteobacteria</b>. Genome Announcements, 5 (32). 8-10.<a href="Resources/Paper PDFs/Crofts et al. - 2017 - Draft Genome Sequences of Three β-Lactam-Catabolizing Soil Proteobacteria.pdf" target="_blank">[PDF]</a></li>

<li><b>Crofts TS*</b>, Gasparrini AJ*, Dantas G (2017) <b>Next-generation approaches to understand and combat the antibiotic resistome</b>. Nature Reviews Microbiology, 15 (7), 422-34.<a href="Resources/Paper PDFs/Crofts, Gasparrini, Dantas - 2017 - Next-generation approaches to understand and combat the antibiotic resistome.pdf" target="_blank">[PDF]</a></li>
</ul>
<h5 align="center">Press</h5>
​<ul>
<li>Bacteria may be powerful weapon against antibiotic resistance
<a href="https://theconversation.com/bacteria-may-be-powerful-weapon-against-antibiotic-resistance-95750" target="_blank">[The Conversation, May 2018 | Terence Crofts]</a></li>

<li>These Bacteria Can Help Fight Antibiotic Resistance
<a href="https://www.sciencefriday.com/segments/these-bacteria-can-help-fight-antibiotic-resistance/" target="_blank">[Science Friday, May 2018 | Katie Hiler]</a></li>

<li>Wash U researchers reveal how some bacteria are able to eat antibiotics
<a href="https://news.stlpublicradio.org/health-science-environment/2018-05-07/wash-u-researchers-reveal-how-some-bacteria-are-able-to-eat-antibiotics" target="_blank">[St. Louis Public Radio, May 2018 | Shahla Farzan]</a></li>

<li>Why some bacteria eat antibiotics and what we can do about it
<a href="https://www.nbcnews.com/health/health-news/why-some-bacteria-eat-antibiotics-what-we-can-do-about-n870181" target="_blank">[NBC news, April 2018 | Maggie Fox]</a></li>

<li>How penicillin-eating bacteria could help design new antibiotics
<a href="https://www.abc.net.au/news/science/2018-05-01/antibiotic-eating-bacteria-resistance-agriculture-penicillin/9710388" target="_blank">[ABC news, April 2018 | Belinda Smith]</a></li>

<li>Bacteria eat antibiotics, and warn each other of threats
<a href="https://cosmosmagazine.com/biology/bacteria-eat-antibiotics-and-warn-each-other-of-threats/#:~:text=The%20researchers%20found%20that%20bacteria%20exposed%20to%20the%20antibiotic%20tobramcyin,the%20effects%20of%20the%20antibiotic." target="_blank">[Cosmos, April 2018 | Paul Biegler]</a></li>

<li>How Bacteria Eat Penicillin
<a href="https://www.the-scientist.com/daily-news/how-bacteria-eat-penicillin-36350" target="_blank">[The Scientist, April 2018 | Shawna Williams]</a></li>

<li>Bacteria’s appetite may be key to cleaning up antibiotic contamination
<a href="https://medicine.wustl.edu/news/bacterias-appetite-may-be-key-to-cleaning-up-antibiotic-contaminated-environments/#:~:text=Bacteria's%20appetite%20may%20be%20key%20to%20cleaning%20up%20antibiotic%20contamination,-Antibiotics%20in%20environment&text=Antibiotics%20in%20the%20environment%20contribute%20to%20drug%20resistance.&text=Louis%20have%20figured%20out%20how,antibiotic%2Dcontaminated%20soil%20and%20waterways." target="_blank">[WUSM news release, April 2018 | Tamara Bhandari]</a></li>

</ul>
</div>

<button type="button" class="collapsible"><h4>Small Molecule Biosynthesis</h4></button>
<div class="content">
<h5 align="center">Description</h5>
<p>In my current position as Research Assistant Professor, working alongside the Kelleher research group, I am pursuing projects that leverage my experience using metagenomics and functional screens to identify and characterize promising biosynthetic gene clusters from microbial communities, including those found in the soil and in the large intestines of animals.
In the past I studied the bacterial biosynthesis of cobamides, co-enzymes structurally related to vitamin B12 that are among the largest and most complex non-polymeric small molecules known. Generally bacteria only produce one cobamide likely due in part to single enzymatic step in biosynthesis that largely determines what form of cobamide is produced. One possible reason for limiting cobamide diversity is that their ability to act as co-enzymes in particular reactions is altered by their structure.</p>
<h5 align="center">Publications</h5>
​<ul>
<li><b>Crofts TS*</b>, Hazra AB*, Tran JLA, Sokolovskaya O, Osadchiy V, Ad O, Pelton J, Bauer S, Taga ME (2014) <b>Regiospecific formation of cobamide isomers is directed by CobT</b>. Biochemistry, 53 (49), 7805-15.<a href="Resources/Paper PDFs/Crofts et al. - 2014 - Regiospecific Formation of Cobamide Isomers Is Directed by CobT.pdf" target="_blank">[PDF]</a></li>

<li><b>Crofts TS</b>, Men Y, Alvarez-Cohen L, Taga ME (2014) <b>A bioassay for the detection of benzimidazoles reveals their presence in a range of environmental samples</b>. Frontiers in Microbiology. 5:592.<a href="Resources/Paper PDFs/Crofts et al. - 2014 - A bioassay for the detection of benzimidazoles reveals their presence in a range of environmental samples.pdf" target="_blank">[PDF]</a></li>

<li>Men Y, Seth EC, Yi S, <b>Crofts TS</b>, Allen RH, Taga ME, Alvarez-Cohen L (2014) <b>Identification of specific corrinoids reveals corrinoid modifications in dechlorinating microbial communities</b>. Environmental Microbiology, 17 (12), 4873-84.<a href="Resources/Paper PDFs/Men et al. - 2014 - Identification of specific corrinoids reveals corrinoid modification in dechlorinating microbial communities.pdf" target="_blank">[PDF]</a></li>

<li><b>Crofts TS</b>, Seth EC, Hazra AB, Taga ME (2013) <b>Cobamide structure depends on both lower ligand availability and CobT substrate specificity</b>. Chemistry and Biology, 20 (10), 1265-74.<a href="Resources/Paper PDFs/Crofts et al. - 2013 - Cobamide structure depends on both lower ligand availability and CobT substrate specificity.pdf" target="_blank">[PDF]</a></li>

<li>Hazra AB, Tran JLA, <b>Crofts TS</b>, Taga ME (2013) <b>Analysis of substrate specificity in CobT homologs reveals widespread preference for 5,6-dimethylbenzimidazole, the lower axial ligand of vitamin B12</b>. Chemistry and Biology, 20 (10), 1275-85.<a href="Resources/Paper PDFs/Hazra et al. - 2013 - Analysis of substrate specificity in CobT homologs reveals widespread preference for DMB, the lower axial ligand o.pdf" target="_blank">[PDF]</a></li>

​</ul>
</div>

<button type="button" class="collapsible"><h4>Microbiome Drug Interactions</h4></button>
<div class="content">
<h5 align="center">Description</h5>
<p>Recent studies have identified the large intestinal microbial community as an important actor in host processing of pharmaceuticals and dietary molecules. This includes the large intestine as a site of medically relevant drug transformations, the interaction of the gut microbiome and host immune system as an important factor in certain cancer treatments, and the gut microbiome as an "organ" vulnerable to damage and dysregulation due to antibiotic treatment. Identification of the microbial taxa responsible for these outcomes, and the corresponding enzymes and mechanisms, could be useful in the identification of individuals with increased risk, allowing for personalized treatment options across multiple arenas of medicine.</p>
<h5 align="center">Publications​</h5>
<ul>
<li><b>Crofts TS</b>, Sontha P, King AO, Wang B, Biddy BA, Zanolli N, Gaumnitz J, Dantas G (2019) <b>Discovery and characterization of a nitroreductase capable of conferring bacterial resistance to chloramphenicol</b>. Cell Chemical Biology. 26 (4). 559-70.<a href="Resources/Paper PDFs/Crofts et al. - 2019 - Discovery and characterization.pdf" target="_blank">[PDF]</a></li>

<li>Keen EC*, <b>Crofts TS*</b>, Dantas G (2018) <b>Checkpoint checkmate: Microbiota modulation of cancer immunotherapy</b>. Clinical Chemistry, 64 (9). 1280-3.<a href="Resources/Paper PDFs/Keen, Crofts, Dantas - 2018 - Checkpoint Checkmate Microbiota Modulation of Cancer Immunotherapy.pdf" target="_blank">[PDF]</a></li>

<li><b>Crofts TS*</b>, Gasparrini AJ*, Dantas G (2017) <b>Next-generation approaches to understand and combat the antibiotic resistome</b>. Nature Reviews Microbiology, 15 (7), 422-34.<a href="Resources/Paper PDFs/Crofts, Gasparrini, Dantas - 2017 - Next-generation approaches to understand and combat the antibiotic resistome.pdf" target="_blank">[PDF]</a>​</li>

<li>Gasparrini AJ*, <b>Crofts TS*</b>, Gibson MK, Tarr PI, Warner BB, Dantas G (2016) <b>Antibiotic perturbation of the preterm infant gut microbiome and resistome</b>. Gut Microbes, 7 (5), 443-9.<a href="Resources/Paper PDFs/Gasparrini et al. - 2016 - Antibiotic perturbation of the preterm infant gut microbiome and resistome.pdf" target="_blank">[PDF]</a></li>

<li>Gibson MK*, <b>Crofts TS*</b>, Dantas G (2015) <b>Antibiotics and the developing infant gut microbiota and resistome</b>. Current Opinion in Microbiology, 27, 51-6.<<a href="Resources/Paper PDFs/Gibson, Crofts, Dantas - 2015 - Antibiotics and the developing infant gut microbiota and resistome.pdf" target="_blank">[PDF]</a></li>
</ul>
</div>

<script>
  var coll = document.getElementsByClassName("collapsible");
  var i;

  for (i = 0; i < coll.length; i++) {
  coll[i].addEventListener("click", function() {
    this.classList.toggle("active");
    var content = this.nextElementSibling;
    if (content.style.display === "block") {
      content.style.display = "none";
    } else {
      content.style.display = "block";
    }
  });
  }

</script>

</section>


<footer>
  <div id="banner">
    <div class="inline-block">
      <a href="https://twitter.com/terencecrofts?lang=en" target="_blank">
        <img alt="Twitter" src="Resources/images/twitter.png" width=50 height=50>
      </a>
    </div>
    <div class="inline-block">
      <a href="https://scholar.google.com/citations?user=3iE1s_wAAAAJ&hl=en" target="_blank">
        <img alt="Google Scholar" src="https://banner2.cleanpng.com/20180503/slq/kisspng-google-scholar-education-research-doctor-of-philos-scholar-s-5aeaddc3af2de5.4479525015253416357175.jpg" width=50 height=50>
      </a>
    </div>
    <div class="inline-block">
      <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Crofts+TS+%5Bauthor%5D" target="_blank">
        <img alt="NCBI Biography" src="https://upload.wikimedia.org/wikipedia/commons/thumb/0/07/US-NLM-NCBI-Logo.svg/829px-US-NLM-NCBI-Logo.svg.png" width=50 height=50>
      </a>
    </div>
  </div>
</footer>

  </body>
</html>
